Add the field Title in Author&Contributors

Edmondi-Kacaj opened the pull request:
Hello everyone,

we saw that the field "Title"  is missing in Author&Contributors, I have added (showing) this field.

Best regards 
Edmond

Edmondi-Kacaj committed:
Add title field for first Author

Edmondi-Kacaj committed:
Merge branch 'develop' of https://github.com/edu-sharing/Edu-Sharing into add-title-for-first-author

torsten-simon commented:
1. Order of Fields should stay identical (1. Given Name, then Surname)
2. Layout should be improved (Title usually doesn't need the same Input space as given/surname)

Edmondi-Kacaj committed:
Merge branch 'develop' of https://github.com/edu-sharing/Edu-Sharing into add-title-for-first-author

 Conflicts:
	Frontend/src/app/common/ui/mds/mds.component.ts

Edmondi-Kacaj committed:
Adjustments in mds:
  - Add new field Title in mds-editor-widget-author.component.html
  - make this field smaller than other

Edmondi-Kacaj committed:
Adjustments in mds "legacy" mode:
  - sort inputs title,firstName,lastName
  - make title input smaller than others

Edmondi-Kacaj commented:
Hello @torsten-simon ,

as you mention in the comment I have made those changes, and the pull request is ready for review.

Edmondi-Kacaj committed:
Merge develop into add-title-for-first-author

Edmondi-Kacaj committed:
fix an merge conflict

Edmondi-Kacaj commented:
Hello @torsten-simon,

when you have time can you please see this pull request.

Edmondi-Kacaj committed:
update core-module

Edmondi-Kacaj committed:
Merge branch 'develop' of https://github.com/edu-sharing/Edu-Sharing into add-title-for-first-author

Edmondi-Kacaj committed:
Merge branch 'develop' of https://github.com/edu-sharing/Edu-Sharing into add-title-for-first-author

Edmondi-Kacaj committed:
Merge branch 'develop' of https://github.com/edu-sharing/Edu-Sharing into add-title-for-first-author
